WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -455,6 +455,12 @@ extension UIImage {
         return UIImage(named: "woo-payments-error")!
     }
 
+    /// Payments plugin
+    ///
+    static var paymentsPlugin: UIImage {
```

Comment:
It would be a good idea to add a test in IconTests, to cover against the possibility of the asset being missing